myapp/config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
-  scope(path_name: { index: '' }) do
-    resources :tasks, path: '/'
-  end
+  root 'tasks#index'
+  resources :tasks
+
+  match '/404', to: 'errors#not_found', via: :all
+  match '/500', to: 'errors#internal_server_error', via: :all
```

Comment:
```suggestion
  get '/404', to: 'errors#not_found'
  get '/500', to: 'errors#internal_server_error'
```
ref. https://railsguides.jp/routing.html#http-verb%E3%82%92%E5%88%B6%E9%99%90%E3%81%99%E3%82%8B